web/packages/teleport/src/Discover/SelectResource/resources.tsx

```diff
@@ -239,9 +262,24 @@ export function getResourcePretitle(r: ResourceSpec) {
       if (r.nodeMeta?.location === ServerLocation.Aws) {
         return 'Amazon Web Services (AWS)';
       }
-      return 'Server';
+      return 'SSH';
     case ResourceKind.SamlApplication:
+      if (
+        r.id === DiscoverGuideId.ApplicationSamlGeneric ||
+        r.id === DiscoverGuideId.ApplicationSamlGrafana
+      ) {
+        return 'Teleport as IDP';
```

Comment:
where does this name show up?

Reply:
it shows up in the `select a resource to enroll tile` :

<img width="338" alt="image" src="https://github.com/user-attachments/assets/119018b3-98dc-41cb-ad8b-43db230cc789" />